user_tools/src/spark_rapids_pytools/resources/emr-configs.json

```diff
@@ -181,7 +181,7 @@
         },
         {
           "resourceKey": "ec2-catalog",
-          "onlineURL": "https://pricing.us-east-1.amazonaws.com/offers/v1.0/aws/AmazonEC2/20230307163705/us-west-2/index.json",
+          "onlineURL": "https://pricing.us-east-1.amazonaws.com/offers/v1.0/aws/AmazonEC2/current/us-west-2/index.json",
```

Comment:
IIUC, this means we are shifting toward a snapshot Vs. using the current/most-recent, is that right?
Do we have to use a snapshot? I would avoid it if it is not necessary because it is very minor details that we can forget about upgrading the snapshot at one time.

Reply:
Yes, that is right. I encountered a failure with the previous snapshot when updating to g5, ref: https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/fetching-price-list-files-manually-step-4.html.